### ISSUE TITLE
Refactor IKeep/KeepBridge

### ISSUE THREAD

**Issue by nkuba** [keep-network/tbtc#221]
As per the comment https://github.com/keep-network/tbtc/pull/204#issuecomment-514782666:

`IKeep` was always intended to be more of a placeholder. There's no sense in clinging to it.

Right now it looks like `KeepBridge` has 3 main purposes:
1. Spin up a new Keep
    1. `requestNewKeep`
2. Translate contract calls from `IKeep` interface to `ECDSAKeep` interface
    1. `getKeepPubkey`
    1. `submitSignatureFraud`
    1. `wasDigestApprovedForSigning`
    1. `distributeEthToKeepGroup`
    1. `distributeERC20ToKeepGroup`
    1. `checkBondAmount`
3. Forward and translate permissioned calls (the really breaking issue)
    1. `approveDigest`
    1. `seizeSignerBonds`
    1. `initialize`

I would suggest that category 1 gets moved to the `TBTCSystem`, and categories 2 and 3 get dropped entirely. There's no reason to have a translation contract for simple calls like `checkBondAmount`. It should be done via a direct reference within the calling contract. Anything else costs additional gas.

So basically the proposal here is to update the existing `Deposit` libraries, and anywhere they use `IKeep(...).ikeep_interface_call()` instead change it so that the use  `ECDSAKeep(...).ecdsakeep_interface_call()` instead.

--
_Implementer_: @nkuba 
_Feature buddy_: @NicholasDotSol
